effekt/shared/src/main/scala/effekt/machine/Transformer.scala

```diff
@@ -168,7 +170,7 @@ object Transformer {
         transform(vargs, bargs).run { (values, blocks) =>
           callee match {
             case Block.BlockVar(id, annotatedTpe, annotatedCapt) =>
-              BPC.info.getOrElse(id, sys.error(s"Cannot find block info for ${id}.\n${BPC.info}")) match {
+              BPC.info.getOrElse(id, sys.error(pp"In ${stmt}. Cannot find block info for ${id}: ${annotatedTpe}.\n${BPC.info}")) match {
```

Comment:
This shouldn't happen, but it does (sadly quite frequently), so I dump a bit of additional info here.